.gitignore

```diff
@@ -5,3 +5,4 @@ _static
 _templates
 /.env
 *.iml
+src/test/resources/example/*_debug.png
```

Comment:
Sure? It might be interesting to see how code changed affect renderings, but I also see that invisible changes are annoying.

Reply:
agreed